kornia/contrib/models/efficient_vit/model.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import torch
+
+from kornia.contrib.models.base import ModelBase
+from kornia.contrib.models.efficient_vit import backbone as vit
+from kornia.core import Tensor
+
+
+@dataclass
+class EfficientViTConfig:
+    """Configuration to construct EfficientViT model.
+
+    Args:
+        checkpoint: URL or local path of model weights.
```

Comment:
to be done next, add here in this docstring the links to the weights available in hf/kornia